vm/class.go

```diff
@@ -1347,22 +1349,19 @@ var builtinClassCommonInstanceMethods = []*BuiltinMethodObject{
 			case 0:
 				return t.vm.initFloatObject(rand.Float64())
 			case 1:
-				_, ok := args[0].(*IntegerObject)
+				err := t.vm.checkArgTypes(args, sourceLine, classes.IntegerClass)
 
-				if !ok {
-					return t.vm.InitErrorObject(errors.TypeError, sourceLine, errors.WrongArgumentTypeFormat, classes.IntegerClass, args[0].Class().Name)
+				if err != nil {
+					return err
 				}
+
 				return t.vm.InitIntegerObject(rand.Intn(args[0].Value().(int)))
 			case 2:
-				_, minOk := args[0].(*IntegerObject)
 
-				if !minOk {
-					return t.vm.InitErrorObject(errors.TypeError, sourceLine, errors.WrongArgumentTypeFormat, classes.IntegerClass, args[0].Class().Name)
-				}
-				_, maxOk := args[1].(*IntegerObject)
+				err := t.vm.checkArgTypes(args, sourceLine, classes.IntegerClass, classes.IntegerClass)
```

Comment:
@lara here's the helper to simplify argument checks I came up with. Let me know your thoughts about it.

Reply:
This is a pretty nice helper especially since it looks like it can check the types of multiple arguments at once. I also love that it's being standardized.